locales/en/allocations.json

```diff
@@ -2,5 +2,13 @@
   "dashboard": {
     "heading": "Allocations",
     "labels": { "single_requests": "single requests" }
-  }
+  },
+  "complete": "Complete",
```

Comment:
Maybe use lowercase for the labels for consistency. 

As before, the label should be read as part of a sentence after the value: `1 complete (allocation)` or `3 incomplete (allocations)`. This doesn't look right if the word is capitalized: `3 Incomplete (allocation)`.